### ISSUE TITLE
NVDA does not announce the pilcrow signs in Microsoft Word

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#11734]
### Steps to reproduce:
1. Open Microsoft Word
2. Type a text
3. Press "Ctrl" + "Shift" + "+"
4. The pilcrow signs appear
5. Read the text with NVDA
6. The NVDA does not announce the pilcrow signs

### Actual behavior:
The NVDA does not announce the pilcrow signs

### Expected behavior:
The NVDA should announce the pilcrow signs and show them on a braille display

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2020.2

#### Windows version:
Edition Windows 10 Home
Version 1909
Operating system build 18363.1016

#### Name and version of other software in use when reproducing the issue:
Microsoft Word 2016
Version 2009 (Build 13231.20262)

#### Other information about your system:
/
#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If addons are disabled, is your problem still occuring?
Did not test.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No.

**Comment by peterpacult** [comment]
The bug still exists; it does not work in Microsoft Word 2016 & 2013 either. The function would be important to keep control over one's own text structure.
